draft-ietf-tls-esni.md

```diff
@@ -112,68 +112,51 @@ comes from {{RFC8446}}; Section 3.
 
 # Overview
 
-This document is designed to operate in one of two primary topologies
-shown below, which we call "Shared Mode" and "Split Mode"
-
-## Topologies
+This document is designed to operate in the topology shown below.
 
 ~~~~
-                +---------------------+
-                |                     |
-                |   2001:DB8::1111    |
-                |                     |
-Client <----->  | private.example.org |
-                |                     |
-                | public.example.com  |
-                |                     |
-                +---------------------+
-                        Server
++----+                   +------------------+    +------------------+
+|    |                   |                  |    |                  |
+|    |                   | pub.example.com  |    | priv.example.com |
+|    |                   | (2001:DB8::1111) |    | (2001:DB8::EEEE) |
+|    |                   |                  |    |                  |
+|    |<---  TLS 1.3  --------------------------->|                  |
+|    |                   |                  |    |                  |
+|    |<-  ESNI Trans-  ->|                  |    |                  |
+|    |     formation     |                  |    |                  |
++----+                   +------------------+    +------------------+
+Client                   Client-Facing Server       Backend Server
 ~~~~
```

Comment:
In most documents, we use "fronting.example.com" and "hidden.example.com". If the space is too tight, you could use "fronting.example" and "hidden.example"

Reply:
Yeah, the length has become even longer, as we have switched from  "fronting" to "client-facing."

490df2c tries to resolve the issue, by wrapping the hostnames.